lib/stm32wb0/CMakeLists.txt

```diff
@@ -0,0 +1,309 @@
+			min(temp ${CFG_BLE_NUM_AUX_SCAN_SLOTS} ${CFG_BLE_NUM_SYNC_SLOTS})
+			MATH(EXPR tasks "${tasks} + ${temp}")
+			if(${CFG_BLE_CONTROLLER_PERIODIC_ADV_WR_ENABLED} AND ${CFG_BLE_CONNECTION_ENABLED})
+				# radio_tasks += 2 * min(NUM_AUX_SCAN_SLOTS_CONF, NUM_SYNC_SLOTS_CONF)
+				min(temp ${CFG_BLE_NUM_AUX_SCAN_SLOTS} ${CFG_BLE_NUM_SYNC_SLOTS})
+				MATH(EXPR tasks "${tasks} + 2*${temp}")
+			endif()
+			if(${CFG_BLE_CONTROLLER_BIS_ENABLED})
+				#radio_tasks += min(NUM_AUX_SCAN_SLOTS_CONF, MAX_NUM_SYNC_BIG)
+				min(temp ${CFG_BLE_NUM_AUX_SCAN_SLOTS} ${CFG_BLE_NUM_SYNC_BIG_MAX})
+				MATH(EXPR tasks "${tasks} +${temp}")
+			endif()
+		endif()
+	endif() # ${CFG_BLE_CONTROLLER_EXT_ADV_SCAN_ENABLED}
+	if(${CFG_BLE_CONTROLLER_CIS_ENABLED} AND ${CFG_BLE_CONNECTION_ENABLED})
+		MATH(EXPR tasks "${tasks} + ${CFG_BLE_NUM_CIG_MAX}")
+	endif()
+endif()
+
+message(STATUS "Number of computed radio tasks: ${tasks}")
+zephyr_compile_definitions(-DCFG_BLE_NUM_RADIO_TASKS=${tasks})
+
+########### End of number of radio tasks calculation ###########
+
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/dm_alloc.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/dtm_cmd_db.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/aci_adv_nwk.c)
+
+set_source_files_properties(BLE_TransparentMode/STM32_BLE/App/dtm_cmd_db.c PROPERTIES COMPILE_FLAGS -Wno-array-bounds)
+set_source_files_properties(STM32_BLE/stack/config/ble_stack_user_cfg.c PROPERTIES COMPILE_FLAGS -Wno-array-parameter)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/Target/bleplat_cntr.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/Target/bleplat.c)
+zephyr_library_sources(BLE_TransparentMode/System/Modules/PKAMGR/Src/pka_manager.c)
+zephyr_library_sources(BLE_TransparentMode/System/Interfaces/hw_aes.c)
+zephyr_library_sources(BLE_TransparentMode/System/Interfaces/hw_pka.c)
+zephyr_library_sources(BLE_TransparentMode/System/Interfaces/hw_rng.c)
+zephyr_library_sources(BLE_TransparentMode/System/Modules/blue_unit_conversion.s)
+zephyr_library_sources(BLE_TransparentMode/System/Modules/osal_memcpy.s)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/adv_buff_alloc_tiny.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/transport_layer.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/dtm_preprocess_events.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/adv_buff_alloc.c)
+zephyr_library_sources(BLE_TransparentMode/System/Modules/RADIO_utils/Src/RADIO_utils.c)
+zephyr_library_sources(BLE_TransparentMode/System/Modules/miscutil.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/pawr_buff_alloc.c)
+zephyr_library_sources(STM32_BLE/stack/config/ble_stack_user_cfg.c)
+zephyr_library_sources(BLE_TransparentMode/STM32_BLE/App/dtm_cmds.c)
+zephyr_library_sources(BLE_TransparentMode/Core/Src/stm32wb0x_hal_msp.c)
+
+else()
+zephyr_compile_definitions(-DCFG_BLE_NUM_RADIO_TASKS=0)
```

Comment:
Do you expect cases where `CONFIG_BT_STM32WB0=y` and `CONFIG_BT=n` ?

Reply:
No, you're right. I'll fix it.